cxx-qt-gen/src/syntax/qtitem.rs

```diff
@@ -10,6 +10,7 @@ use syn::parse::{Parse, ParseStream};
 use syn::{Attribute, Item, ItemMod, Result, Token, Visibility};
 
 #[derive(Clone, PartialEq, Eq)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
why is this suddenly needed? i thought we fixed the 1.63 clippy lints already?

Reply:
I have no idea why clippy decided to complain about this now because nothing in this crate changed. I tried rerunning CI for the last commit in the main branch to double check and it passed, so :shrug:

Reply:
Can we try to remove this ? As locally it passes without it. And i'd prefer the project to have default clippy settings where possible.

Reply:
clippy fails when I comment this out locally

Reply:
Do you have extra lints enabled or something ? As i have `clippy 0.1.63 (4b91a6e 2022-08-08)` and it doesn't...